partiql-tests-data/eval/query/group-by/group-by.ion

```diff
@@ -7094,232 +7191,262 @@
-    assert:{
-      result:EvaluationSuccess,
-      evalMode:[
-        EvalModeCoerce,
-        EvalModeError
-      ],
-      output:$bag::[
-        {
-          categoryId:20,
-          the_count:4,
-          the_agg:1.0
-        },
-        {
-          categoryId:21,
-          the_count:6,
-          the_agg:4.0
-        }
-      ]
-    }
+    assert: [
+      {
+        result:EvaluationSuccess,
+        evalMode: EvalModeCoerce,
+        output:$bag::[
+          {
+            categoryId:20,
+            the_count:4,
+            the_agg:1.0
+          },
+          {
+            categoryId:21,
+            the_count:6,
+            the_agg:4.0
+          }
+        ]
+      },
+      {
+        evalMode:EvalModeError,
+        result:EvaluationFail
+      },
+    ]
   },
   {
     name:"SELECT p.categoryId, COUNT(1) AS the_count, MAX(p.price_mixed) AS the_agg FROM products_sparse AS p GROUP BY p.categoryId",
     statement:"SELECT p.categoryId, COUNT(1) AS the_count, MAX(p.price_mixed) AS the_agg FROM products_sparse AS p GROUP BY p.categoryId",
-    assert:{
-      result:EvaluationSuccess,
-      evalMode:[
-        EvalModeCoerce,
-        EvalModeError
-      ],
-      output:$bag::[
-        {
-          categoryId:20,
-          the_count:4,
-          the_agg:3.0
-        },
-        {
-          categoryId:21,
-          the_count:6,
-          the_agg:5.0
-        }
-      ]
-    }
+    assert: [
+      {
+        result:EvaluationSuccess,
+        evalMode: EvalModeCoerce,
+        output:$bag::[
+          {
+            categoryId:20,
+            the_count:4,
+            the_agg:3.0
+          },
+          {
+            categoryId:21,
+            the_count:6,
+            the_agg:5.0
+          }
+        ]
+      },
+      {
+        evalMode:EvalModeError,
+        result:EvaluationFail
+      },
+    ]
   },
   {
     name:"SELECT p.categoryId, COUNT(1) AS the_count, AVG(p.price_mixed) AS the_agg FROM products_sparse AS p GROUP BY p.categoryId",
     statement:"SELECT p.categoryId, COUNT(1) AS the_count, AVG(p.price_mixed) AS the_agg FROM products_sparse AS p GROUP BY p.categoryId",
-    assert:{
-      result:EvaluationSuccess,
-      evalMode:[
-        EvalModeCoerce,
-        EvalModeError
-      ],
-      output:$bag::[
-        {
-          categoryId:20,
-          the_count:4,
-          the_agg:2.0
-        },
-        {
-          categoryId:21,
-          the_count:6,
-          the_agg:4.5
-        }
-      ]
-    }
+    assert: [
+      {
+        result:EvaluationSuccess,
+        evalMode: EvalModeCoerce,
+        output:$bag::[
+          {
+            categoryId:20,
+            the_count:4,
+            the_agg:2.0
+          },
+          {
+            categoryId:21,
+            the_count:6,
+            the_agg:4.5
+          }
+        ]
+      },
+      {
+        evalMode:EvalModeError,
+        result:EvaluationFail
+      },
+    ]
```

Comment:
So basically all the changed tests from above this point that had used `supplierId_missings`, `supplierId_mixed`, `price_missings`, or `price_mixed` in the `GROUP BY` or aggregation expression which should have resulted in an error in strict mode?